src/app/my-account/my-account.component.html

```diff
@@ -57,9 +57,6 @@ <h2>Your Payment Methods</h2>
       <table id="paymentMethods" *ngIf="paymentMethods !== undefined && paymentMethods.length > 0">
         <tr *ngFor="let method of paymentMethods">
 
-          <!-- We have the card brand twice in the DOM for display in mobile & desktop in different positions. Only one
-          is visible at any time
-          -->
```

Comment:
This comment isn't actually true. It's true about the expiry date not the card brand.